src/main/java/com/ionosenterprise/rest/domain/BackupUnit.java

```diff
@@ -0,0 +1,53 @@
+package com.ionosenterprise.rest.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+public class BackupUnit extends BaseResource {
```

Comment:
javadoc

Reply:
I've followed existing code styles. If the javadoc is necessary a separate ticked will be created for this.